src/components/com_tjcertificate/administrator/includes/tjcertificate.php

```diff
@@ -13,21 +13,37 @@
 
 use Joomla\CMS\MVC\Model\BaseDatabaseModel;
 use Joomla\CMS\Table\Table;
+use Joomla\String\StringHelper;
+use Joomla\CMS\Component\ComponentHelper;
 
 $language = JFactory::getLanguage();
 $language->load('com_tjcertificate');
 
-JLoader::discover("TjCertificate", JPATH_ADMINISTRATOR . '/components/com_tjcertificate/libraries');
-
 /**
  * Certificate factory class.
  *
  * This class creates table and model object by instantiating
  *
  * @since  1.0.0
  */
-class TjCertificateFactory
+class TJCERT
 {
+	/**
+	 * Holds the record of the loaded TJCertificate classes
+	 *
+	 * @var    array
+	 * @since  2.4.0
```

Comment:
@praneettekdi Use the tag to replace the versions

Reply:
ok